add release notes for 11.0.1 (#109)
giantswarm/releases: release-notes/aws/v11.0.1.md

# :zap: Giant Swarm Release 11.0.1 for AWS is now active for you! :zap:

> This release fixes the issue with network traffic between node pools in 10.x and

Reviewer: ```suggestion
This release fixes an issue with network traffic between node pools in 10.x and
```

> 11.x clusters reported yesterday (5th Feb 2020). **To prevent encountering

Reviewer: ```suggestion
11.x clusters reported on the 5th of February 2020. **To prevent encountering
```
